### ISSUE TITLE
Inverted theme.

### ISSUE THREAD

**Issue by AlexDaniel** [bburns/emacswiki-light-theme#12]
The first thing that I noticed is that this theme is very, very bright. Perhaps you could make an inverted variant of it? That is, black background, gray text, everything else could stay the same.

That'd be so cool.

**Comment by bburns** [comment]
Yeah, a couple of people didn't like the brightness, but subthemes like that should be possible - I'll see if I can make a dark version sometime this week, and maybe a dimmer version also.

**Comment by AlexDaniel** [comment]
Oh, there is a dark version now! I like it very much! Thank you.

But here are some of the issues that I have noticed:
![screenshot](https://files.progarm.org/2016-03-09-142112_1058x358_scrot.png)

↑ Here I can barely see the heading. Also, I don't think that the background here was supposed to be light.

![screenshot](https://files.progarm.org/2016-03-09-142243_603x116_scrot.png)
↑ IMO these buttons are too white. It does not irritate me much but it does play nicely with the rest of the theme.

![screenshot](https://files.progarm.org/2016-03-09-142511_653x120_scrot.png)
↑ I am not sure why one of the links is different here.

![screenshot](https://files.progarm.org/2016-03-09-142542_155x57_scrot.png)
↑ Here too.

![screenshot](https://files.progarm.org/2016-03-09-142639_995x506_scrot.png)
↑ If you diff some revisions then you will see this. It is kinda OK but again too bright.

But overall it is very good! I like it!

**Comment by bburns** [comment]
Ah, thanks for all the screenshots - I got swamped recently and haven't had
a chance to do a lot of testing - I'll see if I can fix those this weekend.

**Comment by bburns** [comment]
Hi Alex-Daniel, thanks for your patience - I made the updates to the dark theme, which I'll send on to Alex - please let me know if you notice any other problems. 
Brian

**Comment by AlexDaniel** [comment]
I don't like the way this looks right now:
![screenshot](https://files.progarm.org/2016-03-23-230855_1252x558_scrot.png)

Just a little bit of extra padding makes it better (IMO):
![screenshot](https://files.progarm.org/2016-03-23-231023_1038x298_scrot.png)

Perhaps some inner shadow too:
![screenshot](https://files.progarm.org/2016-03-23-231318_1071x329_scrot.png)

**Comment by AlexDaniel** [comment]
It is also not too good when you hover over the buttons:
![screenshot](https://files.progarm.org/2016-03-23-231728_193x58_scrot.png)
Gray text on gray background – meh! Hard to read.


Now, this is a cool one:
![screenshot](https://files.progarm.org/2016-03-23-232148_1050x532_scrot.png)
↑ Even though the images are kinda gray-ish, they're still shining like a light bulb. There's no such problem when the background is white, but for dark backgrounds you can do this:
```css
filter: brightness(50%);
```
And that's what you get:
![screenshot](https://files.progarm.org/2016-03-23-232415_1050x432_scrot.png)

Of course, don't forget to set brightness back to 100% on ``:hover`` :)

**Comment by AlexDaniel** [comment]
Not sure if it was supposed to be this way:
![screenshot](https://files.progarm.org/2016-03-24-142144_719x170_scrot.png)

**Comment by bburns** [comment]
Good idea with dimming the images - I fixed these and passed the update on to Alex - should be up sometime soon.

**Comment by AlexDaniel** [comment]
This is awesome! Thank you very much!

**Comment by AlexDaniel** [comment]
In terms of contrast this is OK:
![screenshot](https://files.progarm.org/2016-04-02-210752_676x119_scrot.png)
But I don't like the way it looks. But I'm not sure how it can be improved.

----

A bit earlier I recommended this:
```
filter: brightness(50%);
```
It is great, but I've noticed that 50% is a bit too low. Things look unnaturally dark, most of the time even darker than the rest of the page. 70% should be better.
